client/src/partials/templates/modals/findEntity.modal.html

```diff
@@ -1,69 +1,79 @@
-<div class="modal-header">
-  <h4>
-    <i class="glyphicon glyphicon-search"></i>
-    {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
-    <span
-      ng-if="$ctrl.result.uuid"
-      ng-click="$ctrl.refresh()"
-      class="btn btn-default pull-right">
-      <i class="glyphicon glyphicon-refresh"></i>
-    </span>
-  </h4>
-</div>
+<ng-form
+  name="EntityForm"
+  autocomplete="off"
+  autocorrect="off"
+  autocapitalize="off"
+  data-modal="entity"
+  novalidate>
 
-<div class="modal-body clearfix">
+  <div class="modal-header">
+    <h4>
+      <span class="glyphicon glyphicon-search"></span>
+      {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
 
-  <div class="form-group">
+      <span
+        ng-show="$ctrl.result.uuid"
+        ng-click="$ctrl.refresh()"
+        class="btn btn-default btn-sm pull-right">
```

Comment:
No reason that I know of - this was preexisting code, and I mostly just refined it by switching out `ngIf` for `ngShow`/`ngHide`.  We could probably improve this considerably in the future.

Will
```html
<span>left</span>
<span class="text-right">right</span>
```
work?  Or must the `text-align` be on the container?